test/controllers/detailedvaluationrequest/DvrControllerSpec.scala

```diff
@@ -416,6 +422,17 @@ class DvrControllerSpec extends VoaPropertyLinkingSpec {
       case (_, cssClasses) =>
         cssClasses should not include "govuk-tag--grey"
     }
+
+    Inspectors.forAll(checkCaseDetails) { details =>
+      checksTable
+        .getElementsContainingOwnText(details.caseReference)
+        .attr("href") shouldBe s"""/check-case/${details.caseReference}/summary?
+                                  |propertyLinkSubmissionId=${assessments.submissionId}&
+                                  |isOwner=true&
+                                  |valuationId=${assessments.assessments.headOption.fold(
```

Comment:
here, it would be better if you separated out the `${assessments.assessments.headOption.fold(...` into a separate `val` before you dive into string interpolation. Then you could just fit it all on a single line I guess, and would not need the `.stripMargin.replaceAll("\n", "")`